scripts/reindex.py

```diff
@@ -93,17 +99,16 @@ def main(argv: List[str]):
                              prefix=args.prefix,
                              num_workers=args.num_workers,
                              dryrun=args.dryrun)
-    if args.purge:
+    if args.purge or args.index:
         queue_manager = Queues()
         queues = dict(queue_manager.azul_queues())
         # FIXME: eliminate need for filtering (https://github.com/DataBiosphere/azul/issues/1448)
         queues = {k: v for k, v in queues.items() if k in config.work_queue_names}
+    if args.purge:
         logger.info('Disabling lambdas ...')
         queue_manager.manage_lambdas(queues, enable=False)
```

Comment:
PyCharm warns about this line using variables prior to being assigned. Even though we can prove that they will be assigned because `args.purge` implies `args.purge or args.index`, PyCharms static analyzer doesn't catch that. Think about how we can restructure the code so it does not fool PyCharm. Note how the only case in which we don't need the queues is if `args.delete`.

Reply:
In that case it seems overkill to special case the initiation of `queue_manager` and `queues` based on the script arguments and they might as well always be available in the script much like `azul_client`.

Reply:
OK